### ISSUE TITLE
[OpenBMC] autotest/testcase//reventlog/cases0 incorrect

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat-core#3727]
Reviewing the OpenBMC testing... This test case is not quite right... 

```
------START::reventlog_clear::Time:Fri Aug 18 10:21:10 2017------

FILENAME:/opt/xcat/bin/../share/xcat/tools/autotest/testcase//reventlog/cases0

RUN:reventlog p9austin clear [Fri Aug 18 10:21:10 2017]
ElapsedTime:0 sec
RETURN rc = 0
OUTPUT:
Command reventlog is not available now!
CHECK:rc == 0	[Pass]
CHECK:output =~ p9austin\s*:\s*clear|SEL cleared	[Failed]
 
------END::reventlog_clear::Failed::Time:Fri Aug 18 10:21:10 2017 ::Duration::0 sec-----
```

The first check that "Pass" should FAIL.   `Command reventlog is not available now!` 

This should cause FVT to open a bug against reventlog to provide this function or expose it.  Then develpment will have bug to open and expose the API call or track down what is the correct call to run here....  Once it's open, if OpenBMC does not support the endpoint, we should get a HTTP error response.

**Comment by hu-weihua** [comment]
Hi @whowutwut , 
For question ``The first check that "Pass" should FAIL.``. I think the code of test case is right, the expected return code is ``0`` here, but it is a bug of xcat code, it should not return 0.  I agree with you that FVT should open a bug for this.

In fact, @xuweibj and @junxiawang go through the automated test against Austin server everyday, we did not open bug, but they know the reason of the all failed cases. It seems it is not good way. FVT will open all such defect.

**Comment by junxiawang** [comment]
I  open issue:#3745 for this command.

**Comment by hu-weihua** [comment]
Hi @whowutwut, after @xuweibj fix xcat code defect by pr #3967.  This issue has fixed. 

```
------START::reventlog_clear::Time:Thu Sep 21 09:07:03 2017------

RUN:reventlog p9euh01 clear [Thu Sep 21 09:07:03 2017]
ElapsedTime:2 sec
RETURN rc = 0
OUTPUT:
p9euh01: clear
CHECK:rc == 0   [Pass]
CHECK:output =~ p9euh01\s*:\s*clear|SEL cleared [Pass]
```

**Comment by hu-weihua** [comment]
@whowutwut,  I closed this issue first. If you have other problem else, you can reopen it. thanks
